src/main/java/ch/njol/skript/conditions/base/PropertyCondition.java

```diff
@@ -83,14 +95,65 @@ public enum PropertyType {
 		WILL
 	}
 
-	private Expression<? extends T> expr;
+	/**
+	 * @param registry the SyntaxRegistry to register this PropertyCondition with.
+	 * @param condition the class to register
+	 * @param property the property name, for example <i>fly</i> in <i>players can fly</i>
+	 * @param type must be plural, for example <i>players</i> in <i>players can fly</i>
+	 */
+	@ApiStatus.Experimental
+	public static void register(SyntaxRegistry registry, Class<? extends Condition> condition, String property, String type) {
+		register(registry, condition, PropertyType.BE, property, type);
+	}
 
 	/**
+	 * @param registry the SyntaxRegistry to register this PropertyCondition with.
 	 * @param condition the class to register
+	 * @param propertyType the property type, see {@link PropertyType}
 	 * @param property the property name, for example <i>fly</i> in <i>players can fly</i>
 	 * @param type must be plural, for example <i>players</i> in <i>players can fly</i>
 	 */
+	@ApiStatus.Experimental
+	public static void register(SyntaxRegistry registry, Class<? extends Condition> condition, PropertyType propertyType, String property, String type) {
+		if (type.contains("%"))
+			throw new SkriptAPIException("The type argument must not contain any '%'s");
+		SyntaxInfo.Builder<?, ? extends Condition> builder = SyntaxInfo.builder(condition).priority(DEFAULT_PRIORITY);
```

Comment:
Right now, the methods return the SyntaxInfo that was registered. It would be possible to construct a builder from that SyntaxInfo, change the priority, and then register it again. Of course, having to unregister and then register again is not ideal, but needing to change the priority is not exactly common.

An alternative approach could be to revamp these methods to just return a SyntaxInfo (and then you have to register it yourself), but I'm not sure if that's ideal (and it might conflict with the existing method). What do you think?